test/built-ins/Error/cause_property.js

```diff
@@ -0,0 +1,34 @@
+// Copyright (C) 2021 Chengzhong Wu. All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+
+/*---
+description: Error constructor creates own cause property
+info: |
+  Error ( message [ , options ] )
+
+  ...
+  4. Perform ? InstallErrorCause(O, options).
+  ...
+
+  20.5.8.1 InstallErrorCause ( O, options )
+
+  1. If Type(options) is Object and ? HasProperty(options, "cause") is true, then
+    a. Let cause be ? Get(options, "cause").
+    b. Perform ! CreateNonEnumerableDataPropertyOrThrow(O, "cause", cause).
+  ...
+
+esid: sec-error-message
+includes: [propertyHelper.js]
+---*/
+
+var message = "my-message";
+var cause = new Error("my-cause");
```

Comment:
Although we expect developers to set the `cause` to Error instances, there's nothing in the proposal which requires that. Would you mind using a plain Object instead? That will help verify that the feature is value-agnostic without reducing the test's ability to assert identity.